MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+        listView.HeightRequest = extent;
+        loaded = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+N> While loading in `AutoFitMode` make sure that the [ItemSize](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemSize) property value is not specified, to avoid extra space below the list. Update the size of the container after `ListView` loaded to render all the list items in the view.
+
+The following limitations should be noted when using the previous approaches:
+
+* As the entire list items are loaded inside the parent `ScrollView` element, the [ItemAppearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemAppearing) and [ItemDisappearing](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDisappearing) events will not be fired when scrolling.
+* When performing keyboard navigation, the view cannot be scrolled automatically while navigating out of view.
+* Scrolling through the touch action will be handled in all platforms and `SfListView` scrolling will be handled by the parent `ScrollView`.
+
+
+## Working with nested ListView
+ 
+`ListView` allows you to load another `ListView` inside its [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate). When the [AutoFitMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_AutoFitMode) of the outer ListView is `Height`, the size of the inner ListView will be allocated from the maximum screen size. Since the exact size for the inner list cannot be obtained before loading the view. 
+ 
+To get a fixed height for the inner ListView, define a value in its `HeightRequest`. If the items inside the inner ListView are less, allocate the total extend of the inner list to its `HeightRequest`.
+ 
+Follow the steps to set the size for the outer ListView based on the extend of inner ListView:
+
+1. Define a property in the Model class and bind it to the `HeightRequest` of inner ListView, as the height for various inner ListView has to be identified while scrolling.
+2. Hook the [Container](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.VisualContainer.html) `PropertyChanged` event. When the height of the container changes, set the value of `TotalExtent` to the property bound to the `HeightRequest`.
+3. Call the `RefreshView` method asynchronously with few seconds delay in the `Loaded` event of the outer `SfListView`. The height requested for each inner `SfListView` will be set but the outer `SfListView` will not get any notification regarding the size change. So, call the RefreshView method asynchronously after loading the view.

```

Comment:
In the 'Loaded' event of the outer 'SfListView,' call the 'RefreshView' method asynchronously with a few seconds delay. The requested height for each inner 'SfListView' will be set, but the outer 'SfListView' will not get any notification regarding the size change. So, after loading the view, call the RefreshView method asynchronously.

Reply:
I have modified the line as per the concerns provided